code/modules/surgery/organs/tongue.dm

```diff
@@ -310,3 +310,48 @@
 /obj/item/organ/tongue/ethereal/Initialize(mapload)
 	. = ..()
 	languages_possible = languages_possible_ethereal
+
+//Sign Language Tongue - yep, that's how you speak sign language.
+/obj/item/organ/tongue/tied
+	name = "tied tongue"
+	desc = "Cat's got your tounge?"
+	say_mod = "signs"
+	icon_state = "tonguetied"
+	modifies_speech = TRUE
+
+/obj/item/organ/tongue/tied/Insert(mob/living/carbon/M)
+	. = ..()
+	M.verb_ask = "signs"
+	M.verb_exclaim = "signs"
+	M.verb_whisper = "subtly signs"
+	M.verb_sing = "rythmically signs"
+	M.verb_yell = "emphatically signs"
+	ADD_TRAIT(M, TRAIT_SIGN_LANG, "tongue")
+	REMOVE_TRAIT(M, TRAIT_MUTE, "tongue")
+
+/obj/item/organ/tongue/tied/Remove(mob/living/carbon/M, special = 0)
+	..()
+	ADD_TRAIT(M, TRAIT_MUTE, "tongue") //Forces the user to be mute if the tongue is removed, meaning they can't just swap tongues.
```

Comment:
if this is transplanted into someone and then removed they will now be permanently mute

Reply:
Seeing how the only way for one to attain the tongue would be to have the quirk, and the fact that one would need to do organ manipulation to take out their own tongue and put in the tied tongue, I believe it fine. However, if you have another suggestion on how to enforce the trait, it would be appreciated.

Reply:
He handled this